kalite/coachreports/static/js/coachreports/coach_reports/router.js

```diff
@@ -0,0 +1,39 @@
+CoachReportRouter = Backbone.Router.extend({
+    initialize: function(options) {
+        _.bindAll(this);
+        this.state_model = new Backbone.Model();
+        this.listenTo(this.state_model, "change", this.set_url);
+    },
+
+    routes: {
+        "(:facility/)(:group/)(:report/)":    "report"
```

Comment:
Given that the arguments are entirely positional, I am not sure what the url "somegroup/somereport/" would really mean. ```(:facility(/:group(/:report/)))``` is ultimately what is intended, but that is how I read the Backbone Router documentation, that ```(:facility/)(:group/)(:report/)``` means just that, precisely because it parses arguments positionally using '/'.

Absent some identifier, and then using regex to match them, would work, but it would make the route matching unnecessarily complicated. The only reason for making facility optional is so that we can direct to the default facility without it being specified in the referring URL.